tests/features/pages/item-page.feature

```diff
@@ -31,34 +48,19 @@ Feature: item page
   Scenario: One related entity
     When I open `"The Milkmaid" item page`
     Then I see `related entities`
-    And I see the `Painting entity card` in the `related entities`
+    And I see the `Painting related chip` in the `related entities`
 
   Scenario: Multiple related entities
     When I open the `"Het laatste avondmaal" item page`
     Then I see `related entities`
-    And I see the `Leonardo da Vinci entity card` in the `related entities`
-    And I see the `Teodoro Matteini entity card` in the `related entities`
-    And I see the `Engraving entity card` in the `related entities`
+    And I see the `Leonardo da Vinci related chip` in the `related entities`
+    And I see the `Teodoro Matteini related chip` in the `related entities`
+    And I see the `Engraving related chip` in the `related entities`
 
   Scenario: Metadata in another language
     When I open the `"Hammerflügel" item page`
     Then I see a level 1 section heading with the text "Hammerflügel"
 
-  Scenario: Media thumbnail grid for multiple web resources
-    When I open the `"Hammerflügel" item page`
-    And I see the `media thumbnail grid`
-    And the `media thumbnail anchor #1` is marked as "selected"
-    And I click the `media thumbnail anchor #2`
-    Then the `media thumbnail anchor #2` is marked as "selected"
-
-  Scenario: No media thumbnail grid for single web resources
-    When I open `"The Milkmaid" item page`
-    Then I don't have the `media thumbnail grid`
-
-  Scenario: Similar items
```

Comment:
Similar items should still be present on the item page, so should still be tested for.